### DISCUSSION TITLE
aspnetcore-in-action-version-3-update/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#93] [Announcements]
# aspnetcore-in-action-version-3-update/

In this article I talk a little about my upcoming book, ASP.NET Core in Action, Version 3.

https://andrewlock.net/aspnetcore-in-action-version-3-update/

**Comment by acnicholls** [comment]
Where can I find your books?  I’ve been looking for a good structured learning approach to Razor Pages.  I’ve dabbled, but want to learn more.

**Reply by acnicholls** [reply]
Saw the link, nvm

**Comment by jkone27** [comment]
Thank you for your amazing work and book! would it be possible to also cover little sections (or an appendix) for F# language in aspnetcore? thanks again, enjoying your book every time i have some doubt on aspnetcore :)
